runtime/gc_vlhgc/ConfigurationIncrementalGenerational.cpp

```diff
@@ -97,6 +97,22 @@ MM_ConfigurationIncrementalGenerational::createHeapWithManager(MM_EnvironmentBas
 		return NULL;
 	}
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	/* Disable double map if requested page sizes used is equal to huge pages.
+	 * Currently double map is not supported when huge pages are used.
+	 * Note that we keep two double map fields: one for requested and another for
+	 * status. If large pages is enabled, it  will only change the STATUS of double
+	 * mapping, keeping the REQUESTED double mapping field intact.
+	 */
+	if (NULL != heap) {
```

Comment:
not really needed. since we return if NULL prior to even possibly reaching this point.